src/home-assistant.html

```diff
@@ -99,13 +99,13 @@
 
   loadIcons() {
     // If the import fails, we'll try to import again, must be a server glitch
-    // Since HTML imports only resolve once, we import another url.
+    // Since HTML imports only resolve once, in prod mode we import another url.
     const success = () => {
       this.iconsLoaded = true;
     };
 
     Polymer.importHref(
-      '/static/mdi-' + this.icons + '.html',
+      '/static/mdi' + this.icons + '.html',
```

Comment:
Actually this is not necessary. Home Assistant will strip out anything from the url that looks like a hash.

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/http/static.py#L71-L82

Reply:
Oh, I realize now that we don't have fingerprints anymore in dev mode.